introduction.md

```diff
@@ -6,33 +6,31 @@ layout: documentation
 
 # About openHAB
 
-openHAB is **software** for integrating different home automation systems and technologies into one single solution that allows over-arching automation rules, and that offers uniform user interfaces.
+openHAB is **software** for integrating different home automation systems, devices and technologies into one single solution.
+It provides uniform user interfaces, and a common approach to automation rules across the entire system, regardless of the number of manufacturers and sub-systems involved.
 
-But openHAB is also a **community** of users, contributors and maintainers.
+openHAB is also a **community** of users, contributors and maintainers, working together on an open-source, interoperable approach to home automation.
 
 ## openHAB Structure
 
-openHAB 2 is an open-source solution based on the [Eclipse SmartHome](https://eclipse.org/smarthome/) framework. It is written in Java and uses [Apache Karaf](http://karaf.apache.org/) together with [Eclipse Equinox](https://www.eclipse.org/equinox/) as an OSGi runtime and bundles this with [Jetty](https://www.eclipse.org/jetty/) as an HTTP server.
+openHAB 2 is based on the [Eclipse SmartHome](https://eclipse.org/smarthome/) framework, and is written in Java.
```

Comment:
The second part doesn't combine good with the first. How about

> openHAB 2 is developed in Java and mainly based on the [Eclipse SmartHome](https://eclipse.org/smarthome/) framework.

Reply:
Okay - how about, "OH2 is developed in Java and is primarily based on..."?

Reply:
This problem has been resolved.